package.json

```diff
@@ -12,6 +12,9 @@
     "build": "webpack",
     "lint": "resin-lint --typescript src/ test/",
     "test": "npm run lint && JUNIT_REPORT_PATH=report.xml mocha --exit -r ts-node/register -r coffee-script/register -r register-coffee-coverage test/*.{js,coffee} && npm run coverage",
+    "test:fast": "mocha --exit -r ts-node/register -r coffee-script/register test/*.{js,coffee}",
+    "test:build": "tsc && coffee -m -c -o build . && cp -r test/data build/test/ && cp -r src/migrations build/src && cp package.json build",
```

Comment:
I think both `tsc` and `coffee ...` will compile everything under the current directory, even node_modules -- is that intended?

Reply:
`tsc` won't due to the `tsconfig.json`, but I'll check the `coffee` command

Reply:
It doesn't seem to, I guess it adds checks for the `node_modules` directory.